ios/instruments/location_simulation.go

```diff
@@ -0,0 +1,62 @@
+package instruments
+
+import (
+	"strconv"
+
+	"github.com/danielpaulus/go-ios/ios"
+	dtx "github.com/danielpaulus/go-ios/ios/dtx_codec"
+)
+
+const locationSimulationIdentifier = "com.apple.instruments.server.services.LocationSimulation"
+
+// LocationSimulationService gives us access to simulate device geo location
+type LocationSimulationService struct {
+	channel *dtx.Channel
+	conn    *dtx.Connection
+}
+
+// Start sets geolocation with provided params
+func (d *LocationSimulationService) Start(lat string, lon string) error {
```

Comment:
I would be a bit more verbose in the method name, `Start`/`Stop` are very generic. `StartSimulateLocation` and `StopSimulateLocation` are better understandable IMO.